server/src/main/java/org/opensearch/index/seqno/ReplicationTracker.java

```diff
@@ -1208,24 +1215,25 @@ public synchronized void updateVisibleCheckpointForShard(final String allocation
     }
 
     /**
-     * After a new checkpoint is published, start a timer for each replica to the checkpoint.
+     * After a new checkpoint is published, create a timer for each replica to the checkpoint.
      * @param checkpoint {@link ReplicationCheckpoint}
      */
-    public synchronized void setLatestReplicationCheckpoint(ReplicationCheckpoint checkpoint) {
+    public synchronized void setLatestReplicationCheckpoint(ReplicationCheckpoint checkpoint, boolean shouldStartTimers) {
```

Comment:
i don't think we need the boolean flag here - shard can invoke create/start separately?

Reply:
Yep, removed the flag.